[2022-08-22 22:55:05,043] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-22T22:50:00+00:00 [queued]>
[2022-08-22 22:55:05,064] {taskinstance.py:1179} INFO - Dependencies all met for <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-22T22:50:00+00:00 [queued]>
[2022-08-22 22:55:05,065] {taskinstance.py:1376} INFO - 
--------------------------------------------------------------------------------
[2022-08-22 22:55:05,065] {taskinstance.py:1377} INFO - Starting attempt 1 of 1
[2022-08-22 22:55:05,065] {taskinstance.py:1378} INFO - 
--------------------------------------------------------------------------------
[2022-08-22 22:55:05,115] {taskinstance.py:1397} INFO - Executing <Task(PythonOperator): popula_elastic_finalizados> on 2022-08-22 22:50:00+00:00
[2022-08-22 22:55:05,129] {standard_task_runner.py:52} INFO - Started process 106440 to run task
[2022-08-22 22:55:05,138] {standard_task_runner.py:79} INFO - Running: ['***', 'tasks', 'run', 'elastic_desk', 'popula_elastic_finalizados', 'scheduled__2022-08-22T22:50:00+00:00', '--job-id', '1995', '--raw', '--subdir', 'DAGS_FOLDER/etl_elastic_desk.py', '--cfg-path', '/tmp/tmp74u131kk', '--error-file', '/tmp/tmppmh_gje5']
[2022-08-22 22:55:05,140] {standard_task_runner.py:80} INFO - Job 1995: Subtask popula_elastic_finalizados
[2022-08-22 22:55:05,339] {task_command.py:371} INFO - Running <TaskInstance: elastic_desk.popula_elastic_finalizados scheduled__2022-08-22T22:50:00+00:00 [running]> on host 1e1070d9e5ae
[2022-08-22 22:55:05,498] {taskinstance.py:1591} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_EMAIL=clecio.antao@gmail.com
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=elastic_desk
AIRFLOW_CTX_TASK_ID=popula_elastic_finalizados
AIRFLOW_CTX_EXECUTION_DATE=2022-08-22T22:50:00+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2022-08-22T22:50:00+00:00
[2022-08-22 22:55:05,504] {base.py:271} INFO - GET http://192.168.2.100:9200/ [status:200 request:0.004s]
[2022-08-22 22:55:09,598] {base.py:271} INFO - HEAD http://192.168.2.100:9200/_template/desk_finalizados [status:200 request:0.001s]
[2022-08-22 22:55:09,975] {logging_mixin.py:115} WARNING - /home/***/.local/lib/python3.7/site-packages/es_pandas/es_pandas.py:58 RuntimeWarning: invalid value encountered in double_scalars
[2022-08-22 22:55:10,720] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.357s]
[2022-08-22 22:55:11,718] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.131s]
[2022-08-22 22:55:12,620] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.268s]
[2022-08-22 22:55:13,220] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.318s]
[2022-08-22 22:55:14,123] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.068s]
[2022-08-22 22:55:14,406] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.093s]
[2022-08-22 22:55:14,727] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.075s]
[2022-08-22 22:55:15,018] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.099s]
[2022-08-22 22:55:15,439] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.156s]
[2022-08-22 22:55:15,477] {base.py:271} INFO - POST http://192.168.2.100:9200/_bulk [status:200 request:0.183s]
[2022-08-22 22:55:15,500] {logging_mixin.py:115} INFO - 0       22-08-2022
1       22-08-2022
2       22-08-2022
3       22-08-2022
4       22-08-2022
           ...    
4513    23-07-2022
4514    23-07-2022
4515    23-07-2022
4516    23-07-2022
4517    23-07-2022
Name: DataFinalizacao, Length: 4518, dtype: object
[2022-08-22 22:55:15,502] {python.py:173} INFO - Done. Returned value was: None
[2022-08-22 22:55:15,539] {taskinstance.py:1420} INFO - Marking task as SUCCESS. dag_id=elastic_desk, task_id=popula_elastic_finalizados, execution_date=20220822T225000, start_date=20220822T225505, end_date=20220822T225515
[2022-08-22 22:55:15,662] {local_task_job.py:221} WARNING - State of this instance has been externally set to success. Terminating instance.
[2022-08-22 22:55:15,676] {process_utils.py:129} INFO - Sending Signals.SIGTERM to group 106440. PIDs of all processes in the group: [106440]
[2022-08-22 22:55:15,682] {process_utils.py:80} INFO - Sending the signal Signals.SIGTERM to group 106440
[2022-08-22 22:55:15,716] {process_utils.py:75} INFO - Process psutil.Process(pid=106440, status='terminated', exitcode=0, started='22:55:04') (106440) terminated with exit code 0
